cmd/bootstrap/group_replication.go

```diff
@@ -99,26 +105,90 @@ func (m *mysqlsh) runSQL(ctx context.Context, sql string) (string, error) {
 	c.Stdout = &stdoutb
 	c.Stderr = &stderrb
 
-	var result struct {
-		Error string              `json:"error,omitempty"`
-		Rows  []map[string]string `json:"rows,omitempty"`
-	}
+	var result SQLResult
 
 	if err := c.Run(); err != nil {
-		return "", errors.Wrapf(err, "run %s, stdout: %s, stderr: %s", cmd, stdoutb.String(), stderrb.String())
+		return result, errors.Wrapf(err, "run %s, stdout: %s, stderr: %s", cmd, stdoutb.String(), stderrb.String())
 	}
 
 	if err := json.Unmarshal(stdoutb.Bytes(), &result); err != nil {
-		return "", errors.Wrap(err, "unmarshal result")
+		return result, errors.Wrap(err, "unmarshal result")
 	}
 
 	if len(result.Error) > 0 {
-		return "", errors.New(result.Error)
+		return result, errors.New(result.Error)
+	}
+
+	return result, nil
+}
+
+func (m *mysqlsh) getGTIDExecuted(ctx context.Context) (string, error) {
+	result, err := m.runSQL(ctx, "SELECT @@GTID_EXECUTED")
+	if err != nil {
+		return "", err
 	}
 
 	return result.Rows[0]["@@GTID_EXECUTED"], nil
 }
 
+func (m *mysqlsh) getGroupSeeds(ctx context.Context) (string, error) {
+	result, err := m.runSQL(ctx, "SELECT @@group_replication_group_seeds")
+	if err != nil {
+		return "", err
+	}
+
+	return result.Rows[0]["@@group_replication_group_seeds"], nil
+}
+
+func (m *mysqlsh) setGroupSeeds(ctx context.Context, seeds string) (string, error) {
+	sql := fmt.Sprintf("SET GLOBAL group_replication_group_seeds = \"%s\"", seeds)
```

Comment:
Should we maybe use SET PERSIST? What will happen if node gets restarted, if I understand it will update peer variables, but what will be the content of this variable of the restarted pod?